searchcore/src/vespa/searchcore/proton/server/documentdbconfigmanager.cpp

```diff
@@ -382,6 +398,10 @@ DocumentDBConfigManager::update(const ConfigSnapshot &snapshot)
         (*new_threading_service_config == *old_threading_service_config)) {
         new_threading_service_config = old_threading_service_config;
     }
+    auto new_alloc_config = build_alloc_config(_bootstrapConfig->getProtonConfig(), _docTypeName);
```

Comment:
Consider if this change should be tested in src/tests/proton/proton_config_fetcher/proton_config_fetcher_test.cpp.